src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
+  const newSeparator = toFormat.slice(3);
+  const splitDate = date.split(oldSeparator);
```

Comment:
You are splitting the date using `oldSeparator` which is the result of an incorrect slice operation. After fixing the `oldSeparator`, use it correctly to split the date string.